xchange-ftx/src/main/java/org/knowm/xchange/ftx/service/FtxLendingService.java

```diff
@@ -0,0 +1,277 @@
+package org.knowm.xchange.ftx.service;
+
+import org.apache.commons.lang3.StringUtils;
+import org.knowm.xchange.ftx.FtxExchange;
+import org.knowm.xchange.ftx.dto.account.*;
+import org.knowm.xchange.utils.BigDecimalUtils;
+
+import java.io.IOException;
+import java.math.BigDecimal;
+import java.util.*;
+import java.util.stream.Collectors;
+
+public class FtxLendingService extends FtxBaseService {
```

Comment:
Can you call this and borrowing FtxLendingServiceRaw + FtxBorrowingServiceRaw to keep in line with current pattern. Ideally we will end up having BorrowingService and LendingService classes

Reply:
No probleme